src/Components/ChangeRegionModal.js

```diff
@@ -1,35 +1,111 @@
 import React, { Component } from 'react';
-import { MDBContainer, MDBBtn, MDBModal, MDBModalBody, MDBModalHeader, MDBModalFooter } from 'mdbreact';
+import { MDBContainer, MDBBtn, MDBModal, MDBModalBody, MDBModalHeader, MDBModalFooter, MDBRow, MDBCard } from 'mdbreact';
 import './styles/ChangeRegionModal.css'
+const locations = require('../commons/locations.json');
+const location_id = ["non", "Seattle", "San Francisco / Bay", "Los Angeles", "Silicon Valley", "Chicago", "Boston", "New York", "Austin", "Vancouver", "Calgary", "Edmonton", "Winnipeg", "Kitchener-Waterloo", "Toronto", "Peel Region", "York Region", "Ottawa", "Montreal", "Quebec City"]
```

Comment:
See comments from below in `locations.js` . We can make this more maintainable if we directly reference a discrete set of locations only from the json file, rather than maintaining a separate array of keys.

Reply:
The indices are to map the id to the buttons to know which is active.
Should I add another field in the locations.json for the id # or

Reply:
Resolved in refactor.